sandbox-template.md

```diff
@@ -0,0 +1,162 @@
+# CNCF SIG Sandbox Project Request Template & Promotion Template
```

Comment:
Can we make this merged with project proposal requriements? 

Here is the previous proposal requirements: https://github.com/cncf/toc/blob/master/process/project_proposals.adoc#project-proposal-requirements.

Here is a set of proposals: https://github.com/cncf/toc/tree/master/proposals

So project maintainers only need to create one documentation covering both the SIG needs and the general info as a proposal record.